lib/tasks/publishing_api.rake

```diff
@@ -12,19 +12,6 @@ namespace :publishing_api do
     end
   end
 
-  desc "Publish pre-production finders only, to the Publishing API"
```

Comment:
This removal should probably be fixed up to the "Remove usages of pre_production flag" commit.

Reply:
But the logic is switched and changed in this commit, so this rake task would be broken anyways - we thought it'd be best to remove in this commit